Library/Formula/libdecnumber.rb

```diff
@@ -0,0 +1,41 @@
+class Libdecnumber < Formula
+  desc "ANSI C implementation of General Decimal Arithmetic"
+  homepage "http://speleotrove.com/decimal/#decNumber"
+  url "http://download.icu-project.org/files/decNumber/decNumber-icu-368.zip"
+  version "3.68"
+  sha256 "14ec2cf30b58758493a7661b78b80abfb281652b61a425b85cda83173518fe25"
+  resource "cmakelists" do
```

Comment:
Ok lets take this one point at a time:

* unsupported, unknown-by-upstream build steps. You suggestion to write this in Ruby does not change anything here, except that we would do these steps in an unsuitable language. In addition I do not agree on the unsupported claim. If there are no build steps whatsoever, any build steps are supported. It seems that you are suggesting the opposite, that is none are supported. So you claim this library is unbuildable and unusable, which is not very constructive.
* lack of convenient build process: upstream provides **no** build process. It offers instructions on how to build the examples in readme.txt and no instructions in how to build a shared or static library. Because of this we as the owners of this formula are in fact the owners of the build process. This means writing our own build scripts. Choosing Ruby over CMake for build scripts is IMO unwise.

I can try to sell the build script upstream but given how infrequently this library is updated (stable since 2010) I wouldn't want to hold my breath for a new release. That also reminds me that in [errata](http://speleotrove.com/decimal/decnumerr.html) there is a fix (in text) for a bug in the latest release. I am sure you will agree that chances of a CMakeLists making it into any release are slim.